File: submissions/majority-element.py
// https://leetcode.com/problems/majority-element

class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        from collections import Counter
        floor = len(nums) // 2
        x = Counter(nums)
        for i in x:
            if x[i] > floor:
                return i

File: submissions/majority-element.go
// https://leetcode.com/problems/majority-element

func majorityElement(nums []int) int {
    maj := 0
    count := 0
    for _, ele := range nums {
        if count == 0 {
            maj = ele
            count = 1
        } else if ele == maj {
            count++
        } else {
            count--
        }
    }

    return maj
}